### PR TITLE
added ability to not include PTR based on an argument

### PR DESCRIPTION
**Pull request by Topener**

### CODE REVIEW

File: nl.fokkezb.pullToRefresh/controllers/widget.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ $.getControl = getControl;
 
 (function constructor(args) {
 
-  if (!OS_IOS && !OS_ANDROID) {
-    console.warn('[pullToRefresh] only supports iOS and Android.');
+  if (args.noShow || (!OS_IOS && !OS_ANDROID)){
```
Comment (FokkeZB) [inline nl.fokkezb.pullToRefresh/controllers/widget.js]: Mmm, maybe `hidden`?

**Comment by Topener** [comment]
I renamed it to hidden. It might be confusing naming (both this one and the previous one) because it is not actually hidden, it is never initialised. But I guess that speaks for itself

**Comment by nuno** [comment]
I do not like the "hidden" to be honest!
Some name:  "stopInit", "notAutoInit", "blockInit", "blockAutoInit" ...

**Comment by FokkeZB** [comment]
Mmm, it might be confusing with the widget's `hide()` method.

Actually Rene, the way you coded it the table/listview the widget wraps will also not display if you use the new argument because https://github.com/Topener/nl.fokkezb.pullToRefresh/blob/28fa80d66a1766932516ec2280451f5e57e75d07/nl.fokkezb.pullToRefresh/controllers/widget.js#L17 never runs.

So either you need to still run the above line (and stop after that) or you can simply achieve the same effect by using Alloy's `if` attribute.

**Comment by Topener** [comment]
Yeah the reason for this functionality to be added was actually to prevent PTR from being added, not hide it. So... it is terminology.

**Comment by Topener** [comment]
Adding `if` doesn't work because then it wouldn't render the ListView either as PTR is a parent of the ListView

**Comment by Topener** [comment]
Ok changed name a final time to `dontInit`. I guess that solves any confusion?

**Comment by FokkeZB** [comment]
Hey, you are the boss now! I'd just suggest to also document the new arg in the readme.

File: nl.fokkezb.pullToRefresh/controllers/widget.js [hunk 2]
```diff
@@ -10,8 +10,8 @@ $.getControl = getControl;
 
 (function constructor(args) {
 
-  if (!OS_IOS && !OS_ANDROID) {
-    console.warn('[pullToRefresh] only supports iOS and Android.');
+  if (args.dontInit || (!OS_IOS && !OS_ANDROID)){
+    if (!args.dontInit) console.warn('[pullToRefresh] only supports iOS and Android.');
```
Comment (nuno) [inline nl.fokkezb.pullToRefresh/controllers/widget.js]: @Topener I suggest add something in that console.warn() that help reflect "args.dontInit". But that is my opinion, of course.

**Pull request merged**